company-box.el

```diff
@@ -355,6 +355,9 @@ Examples:
     (set-window-dedicated-p window t)
     (redirect-frame-focus frame (frame-parent frame))
     (set-frame-parameter frame 'name "")
+    (with-selected-frame frame
+      (when (bound-and-true-p tab-bar-mode)
+        (tab-bar-mode -1)))
```

Comment:
This does not work as expected, as it would also turn off tab-bar-mode for main frame. 
Instead of `(tab-bar-mode -1)`, we can use `(set-frame-parameter frame 'tab-bar-lines 0)`